llpc/tool/amdllpc.cpp

```diff
@@ -1497,11 +1497,12 @@ static Result processPipeline(ICompiler *compiler, ArrayRef<std::string> inFiles
 }
 
 #ifdef WIN_OS
-// =====================================================================================================================
-// Finds all filenames which can match input file name
-void findAllMatchFiles(const std::string &inFile,           // [in] Input file name, include wildcard
-                       std::vector<std::string> *pOutFiles) // [out] Output file names which can match input file name
-{
+/// ====================================================================================================================
+/// Finds all filenames which can match input file name
+///
+/// inFile [in]     : Input file name, including a wildcard.
```

Comment:
For non-interface files (apart from vkgc.h, llpc.h), the parameter comments:
`// @param [out] paramVar : Parameter comments`
Only add [out] and [in,out] tag. No [in] tag.
You can refer existing codes. I see this issue in another your PR.

Reply:
Thanks. I wish there was a clang-format/tidy check for this. I always forget what the format is when I switch between projects.